### ISSUE TITLE
validation accuracy at 50% instead of 60%?

### ISSUE THREAD

**Issue by polo5** [kratzert/finetune_alexnet_with_tensorflow#50]
I get 50% accuracy on the ILSVRC2012 validation set without training, which falls short of the 60% boasted in the [AlexNet paper](https://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf) (i.e 40.7% top-1 error rate).

Any idea what could be the problem? I'm thinking this could be due to the parameters of the local_response_normalisation layer: yours seem different than the caffe AlexNet [here](https://github.com/BVLC/caffe/blob/master/models/bvlc_alexnet/train_val.prototxt). For instance you have alpha=1e-5 instead of 1e-4. I've been playing with those but have yet to find a configuration that gets me to 60%.

**Comment by kratzert** [comment]
thanks for the hint with the normalization factor. I think I remember that I put it for some reason to 1e-5 but it is so long ago and I coudn't find a commit where this was discussed. So I changed it to the publication value again. Regarding the ImageNet validation: I never tried it actually. One thing, while accuracy could probably not match is the difference in image loading ( See [this issue](https://github.com/kratzert/finetune_alexnet_with_tensorflow/issues/15)). Another reason might be, that you test on single images. In the original AlexNet paper, they worked with 5 random crops from the original 256x256 images and averaged the predictions.
If you find other reasons, why the accuracy may be different, let me know.

**Comment by zhaoyanlyu** [comment]
Great thanks to @polo5 and, of course, to @kratzert too. The model with 1e-5 alpha gives 23% error rate on validation set of ILSVRC2012, which is close enough. Other tricks mentioned by @kratzert might improve a couple percentage as well.
